Libraries/LibWeb/CSS/CSSImportRule.cpp

```diff
@@ -76,7 +77,9 @@ String CSSImportRule::serialized() const
     if (m_supports)
         builder.appendff(" supports({})", m_supports->to_string());
 
-    // FIXME: 3. If the rule’s associated media list is not empty, a single SPACE (U+0020) followed by the result of performing serialize a media query list on the media list.
+    // 3. If the rule’s associated media list is not empty, a single SPACE (U+0020) followed by the result of performing serialize a media query list on the media list.
+    for (auto& media_query : m_media_query_list)
+        builder.appendff(" {}", media_query->to_string());
```

Comment:
This doesn't put a comma between the media queries. There's a `String serialize_a_media_query_list(Vector<NonnullRefPtr<MediaQuery>> const&);` in MediaQuery.h